redesign/simple_server.js

```diff
@@ -0,0 +1,14 @@
+var liveServer = require("live-server");
+
+var params = {
+    port: 8181,
+    host: "0.0.0.0",
+    root: "./redesign",
+    open: false,
+    ignore: 'scss,my/templates',
+    file: "./index.html",
+    wait: 1000,
+    mount: [['/components', './node_modules']],
+    logLevel: 2,
+    middleware: [function(req, res, next) { next(); }]
```

Comment:
I think we can simplify the list of parameters. Do we need to define all of them? For example ```ignore: 'scss, my/templates'" doesn't match any files in the new website. I'm not sure about the ```mount: [['/components', './node_modules']]``` property do we need it (I'll check the doc of live-server)